webpack/common.js

```diff
@@ -63,14 +63,16 @@ exports.getRules = function(config) {
 			test: /\.(png|jpe?g|gif|svg)(\?.*)?$/,
 			type: 'asset/resource',
 			generator: {
-				filename: 'assets/svg/[name][contenthash:8][ext]'
+				filename: 'svg/[name][contenthash:8][ext]',
+				outputPath: mode === 'dev' ? '' : 'assets/'
```

Comment:
@petrKavulok Not sure if this will not lead to a bigger confusion, as from my subjective point of view, I would keep `outputPath` separated and not mixing it in the common file. (But I understand your intention)